### DISCUSSION TITLE
Query on database after auth.getUser() goes wrong.

### DISCUSSION THREAD

**Discussion by mvrochar** [adonisjs/core#2176] [Help]
When I try to use auth,getUser() on controller, like so:

`async index({auth}) {
    try{
      await auth.getUser().then(response => console.log(response)).catch(err => console.log(err))
    }catch(err){
      console.log(err)
    }
    return await MainModel.all()
  }`

I get the following error from mysql:

![image](https://user-images.githubusercontent.com/49122424/105848409-b0869600-5fbd-11eb-9d38-a57a3615cbf1.png)

I know why it occurs(the correct column name is UserId), but I have no Idea where to change the query.

Any help? Thanks in advance

**Comment by wfelipe99** [comment]
If I remember correctly for some methods to work properly, you must have a `id` property in your model. If you want to have a different column name in your table, you can to this:

```javascript
export default class User extends BaseModel {
  // This decorator indicates that Adonis should look for "UserId" instead of "id" in the table Users.
  // It'll be used when you run your migration too. So, it'll create a column named "UserId".
  // So when you use a method like .find(), Adonis will look for "UserId" instead of "id" in the table.
  @column({ isPrimary: true, columnName: 'UserId' })
  public id: number
}
```

**Reply by mvrochar** [reply]
Thank you! Will try that
